### PR TITLE
DOC: Add example to stats.normaltest

### PR DESCRIPTION
**Pull request by MartinThoma**

### CODE REVIEW

File: scipy/stats/stats.py [hunk 1]
```diff
@@ -1394,6 +1394,20 @@ def normaltest(a, axis=0, nan_policy='propagate'):
     .. [2] D'Agostino, R. and Pearson, E. S. (1973), "Tests for departure from
            normality", Biometrika, 60, 613-622
 
+    Examples
+    --------
+    >>> import scipy.stats as stats
```
Comment (WarrenWeckesser) [inline scipy/stats/stats.py]: `from scipy import stats` is simpler. (["Don't repeat yourself"](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself); why write `stats` twice?)

File: scipy/stats/stats.py [hunk 2]
```diff
@@ -1394,6 +1394,20 @@ def normaltest(a, axis=0, nan_policy='propagate'):
...
+    >>> pts = 1000
+    >>> a = np.random.multivariate_normal([0], [[1]], size=pts)
```
Comment (WarrenWeckesser) [inline scipy/stats/stats.py]: This uses `np.random.multivariate_normal()`, but it generates a one-dimensional sample, so you might as well use `np.random.normal(0, 1, size=pts)` instead.  (The same comment applies to the next line.)

File: scipy/stats/stats.py [hunk 3]
```diff
@@ -1394,6 +1394,20 @@ def normaltest(a, axis=0, nan_policy='propagate'):
...
+    >>> b = np.random.multivariate_normal([2], [[1]], size=pts)
+    >>> x = np.concatenate((a, b))
+    >>> k2, p = stats.normaltest(x)
+    >>> alpha = 10**-3
```
Comment (WarrenWeckesser) [inline scipy/stats/stats.py]: It would be more conventional to write this as `alpha = 1e-3`.

File: scipy/stats/stats.py [hunk 4]
```diff
@@ -1394,6 +1394,20 @@ def normaltest(a, axis=0, nan_policy='propagate'):
...
            normality", Biometrika, 60, 613-622
 
+    Examples
+    --------
+    >>> import scipy.stats as stats
+    >>> pts = 1000
+    >>> a = np.random.multivariate_normal([0], [[1]], size=pts)
+    >>> b = np.random.multivariate_normal([2], [[1]], size=pts)
+    >>> x = np.concatenate((a, b))
+    >>> k2, p = stats.normaltest(x)
```
Comment (WarrenWeckesser) [inline scipy/stats/stats.py]: In addition to the test below, I'd like to see the p-value.  How about adding `print(p)` after this?

**Review by WarrenWeckesser — CHANGES_REQUESTED** [review]
Thanks for the example!  I've made some suggestions in-line.

File: scipy/stats/stats.py (hunk 4 above)
Comment (MartinThoma) [inline scipy/stats/stats.py]: `print(p)` is a problem due to doctests as the value differs between runs

Comment (WarrenWeckesser) [inline scipy/stats/stats.py]: One way to avoid that is to set the random seed with, say, `np.random.seed(123)` before generating the random samples.

Comment (WarrenWeckesser) [inline scipy/stats/stats.py]: I see that you added a call to set the seed.  Were you also going to add `print(p)`?

**Comment by MartinThoma** [comment]
I print p in line 1408

**Comment by WarrenWeckesser** [comment]
> I print p in line 1408

Ah, sorry I missed that.  But that only shows `p` in one of the cases, and with that formatting, it shows all zeros.  As an example of `normaltest()`, I think it would be (slightly!) better to show the actual value of `p` in either case.  How about `print("p = {:g}".format(p))` or `print("p = %g" % p)` before the `if` statement?

**Review by WarrenWeckesser — APPROVED** [review]

**Comment by WarrenWeckesser** [comment]
Thanks @MartinThoma, merged.

**Pull request merged**